test/new-e2e/tests/agent-metric-logs/log-agent/listener/tcp_test.go

```diff
@@ -0,0 +1,112 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
I only see a `tcp_test.go` here. It's not clear from the readme if there should also be a `udp_test` as well at this point. I understand this is kind of a WIP - but just wanted to double check if a file was missed.

Reply:
Good observation! I had the Docker Compose file ready to go when I found that TCP/UDP wasn't supported in a container environment. So yeah, no `udp_test` at the moment :/